challenges.yml

```diff
@@ -1699,3 +1699,448 @@ extending_smart_watcher_2:
           },
           ```
       correct: a
+
+extending_css_transitions:
+  question: |
+    Imagine I have a simple component that can show a message when one of our
+    customers have bought too much pizza.
+
+    ```vue
+    <template>
+        <div :class="{
+                [$style.component]: true,
+                [$style.tall]: showDisclaimer,
+            }"
+        >
+            <button @click="addPizzaToCart" />
+
+            <p v-show="showDisclaimer">
+                <strong>Disclaimer</strong>: Eating too much pizza can be
+                bad for your help!
+            </p>
+        </div>
+    </template>
+
+    <script>
+    import { buyPizza } from '@/services/buy-pizza';
+
+    export default {
+        name: 'PizzaCheckout',
+        data() {
+            return {
+                showDisclaimer: false,
+            };
+        },
+        methods: {
+            async addPizzaToCart() {
+                await buyPizza();
+            }
+        },
+    };
+    </script>
+    ```
+
+    In a Single Page Application, where this component has a fixed height, I want to
+    add some styling so that its height will *transition* smoothly when that message
+    toggles on and off.
```

Comment:
I think the "Single Page Application" part is not important, or is it?

Reply:
You are right, may be not!